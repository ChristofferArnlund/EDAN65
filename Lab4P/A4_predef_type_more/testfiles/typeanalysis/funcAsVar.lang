int l() {

}

int m() {
	int k = l;
}
