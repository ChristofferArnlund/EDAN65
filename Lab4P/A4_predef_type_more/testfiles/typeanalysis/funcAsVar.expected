Error at line 6: 'l' is not a variable
